.github/workflows/test_and_build.yml

```diff
@@ -46,3 +46,40 @@ jobs:
 
     - name: Build package
       run: poetry build
+
```

Comment:
`publish_to_dockerhub` will be moved to `release.yml` later, right now I just want to test it in the PR.